### ISSUE TITLE
Density maps availability

### ISSUE THREAD

**Issue by PolarBean** [yxu233/Xu_Bergles_brainwide_oligo_map#1]
Cool project, I am wondering whether you will also share the density maps?

**Comment by tigeryuxu** [comment]
Hey! Sorry for the delayed reply, not used to checking Github. But yes, absolutely, that's a great idea, we can share them.

If you have a specific map youre interested in I can send it right away, otherwise, we will just share everything altogether at the end. We're still working on uploading and hosting the rest of the raw data.

**Comment by PolarBean** [comment]
No worries! I will wait for them to be uploaded together,  thanks for your response

**Comment by PolarBean** [comment]
You could consider sharing it on the EBRAINS data repository (you can see it here https://search.kg.ebrains.eu). It's possible to share large files and include neuroglancer links https://www.ebrains.eu/data/share-data/ 
It could be a good way to make sure all the data is publicly available.
(Full disclosure my PhD is partially funded by EBRAINS and I've contributed to several datasets.)

**Comment by tigeryuxu** [comment]
Oh cool! Thanks for the info, will definitely check it out. I think we're currently setting up to host with BossDB but will compare to see what is best.

**Comment by PolarBean** [comment]
yeah BossDB also seems good, you may also be interested in displaying your Density maps in Siibra. its built on Neuroglancer but designed specifically to view atlas registered data. 
[here's a density map ](https://atlases.ebrains.eu/viewer/#/a:juelich:iav:atlas:v1.0.0:2/t:minds:core:referencespace:v1.0.0:265d32a0-3d84-40a5-926f-bf89f68212b9/p:minds:core:parcellationatlas:v1.0.0:05655b58-3b6f-49db-b285-64b5a0276f83/rn:f255401/@:0.0.0.-W000..2xhyO8.2-9sA4.-GQZj.2yO5eB..mJN..A3v2~.9kCA.4Yv..89-/x-overlay-layer:precomputed:%2F%2Fhttps:%2F%2Fdata-proxy.ebrains.eu%2Fapi%2Fv1%2Fbuckets%2Fd1-d2-expression%2Fd1_ng)I created for a previous project viewed with Siibra (it's the distribution of cells that express the d1 receptor). Toggle the opacity to 0.5 so you can see the underlying template. Once your density maps have neuroglancer links it's pretty trivial to generate a Siibra link (I'd be happy to do it).
